plugins/org.locationtech.udig.catalog/src/org/locationtech/udig/catalog/internal/ServiceFactoryImpl.java

```diff
@@ -19,114 +19,112 @@
 import org.locationtech.udig.catalog.IService;
 import org.locationtech.udig.catalog.IServiceFactory;
 import org.locationtech.udig.catalog.ServiceExtension;
 import org.locationtech.udig.catalog.interceptor.ServiceInterceptor;
 
-import org.eclipse.core.runtime.IProgressMonitor;
-import org.eclipse.core.runtime.IStatus;
-import org.eclipse.core.runtime.NullProgressMonitor;
-import org.eclipse.core.runtime.Status;
-import org.eclipse.core.runtime.SubProgressMonitor;
-
 /**
  * Default implementation of IServiceFactory used by the local catalog.
  * <p>
- * This service factory is careful not to test the connections (ie resolve to IServiceInfo)
- * during creation. It is making the assumption that the parameters were already checked
- * (and are thus valid), the resulting IService handles are thus untested and not connected
- * as behoves the application when quickly starting up.
- * 
+ * This service factory is careful not to test the connections (ie resolve to IServiceInfo) during
+ * creation. It is making the assumption that the parameters were already checked (and are thus
+ * valid), the resulting IService handles are thus untested and not connected as behaves the
+ * application when quickly starting up.
+ *
  * @author David Zwiers, Refractions Research
  * @since 0.6
  * @version 1.3
  */
 public class ServiceFactoryImpl extends IServiceFactory {
 
-
-
     /** @deprecated use createService */
-    public List<IService> aquire( final URL id, final Map<String, Serializable> params ) {
+    @Deprecated
+    @Override
+    public List<IService> aquire(final URL id, final Map<String, Serializable> params) {
         return createService(params);
     }
+
     /** @deprecated use createService */
-    public List<IService> aquire( Map<String, Serializable> params ) {
+    @Deprecated
+    @Override
+    public List<IService> aquire(Map<String, Serializable> params) {
         return createService(params);
     }
+
     /** @deprecated use createService */
-    public List<IService> acquire( Map<String, Serializable> params ) {
+    @Deprecated
+    @Override
+    public List<IService> acquire(Map<String, Serializable> params) {
         return createService(params);
     }
+
     /** @deprecated use createService */
-    public List<IService> aquire( final URL target ) {
+    @Deprecated
+    @Override
+    public List<IService> aquire(final URL target) {
         return createService(target);
     }
+
     /** @deprecated use createService */
-    public List<IService> acquire( URL target ) {
+    @Deprecated
+    @Override
+    public List<IService> acquire(URL target) {
         return createService(target);
     }
 
     /** @deprecated use createService */
-    public List<IService> acquire( final URL id, final Map<String, Serializable> params ) {
+    @Deprecated
+    @Override
+    public List<IService> acquire(final URL id, final Map<String, Serializable> params) {
         return createService(params);
     }
```

Comment:
Is there another pull request that removes deprecated aquire methodes from IServiceFactory?

Reply:
I have a branch which is doing this action but I stopped the PR as I need to sort things out!